multimap/src/main/java/org/infinispan/multimap/impl/ExternalizerIds.java

```diff
@@ -0,0 +1,6 @@
+package org.infinispan.multimap.impl;
+
+public interface ExternalizerIds {
+   Integer PUT_KEY_VALUE_FUNCTION = 3001;
```

Comment:
Do you mean update the doc or update the numbers ? 
ClusteredCounters range are 2000-2011. I was assuming I can add another range starting on 3...